source/dotnet/Test/AdaptiveCards.Test/AllPayloadTests.cs

```diff
@@ -97,12 +97,11 @@ private void TestPayloadsInDirectory(string path, string[] excludedCards)
             }
         }
 
-        [TestMethod]
-        public void TestAllScenarios()
-        {
-            TestPayloadsInDirectory(Path.Combine(SamplesPath, "v1.0", "scenarios"), null);
-            TestPayloadsInDirectory(Path.Combine(SamplesPath, "v1.1", "scenarios"), null);
-        }
+        // [TestMethod]
```

Comment:
I don't think .NET supports v1.5, so I commented out these tests.

@paulcam206 @almedina-ms is this the correct behavior?

Reply:
v1.5 is code complete for dot net with two PR open.

Reply:
I created an issue to address this once .NET is ready for v1.5 - #8071